index.html

```diff
@@ -2930,6 +2931,40 @@ <h2 id="View">Backbone.View</h2>
       events that the view has <a href="#Events-listenTo">listenTo</a>'d.
     </p>
 
+    <p id="View-events">
+      <b class="header">events</b><code>view.events or view.events()</code>
+      <br />
+      The <b>events</b> hash (or function) can be used to specify DOM event listeners
+      for your View. By explicitly defining the <b>events</b> property of your View,
+      Backbone will automatically attach the event listeners at instantiation time,
+      since it attempts to <a href="#View-delegateEvents">delegateEvents</a> right
+      before invoking <a href="#View-constructor">initialize</a>.
+    </p>
+
+    <p>
+      This can also be a function that returns an <b>events</b> hash.
+    </p>
+
+<pre>
+var InputView = Backbone.View.extend({
+
+  tagName: 'input',
+
+  events: {
+    "keydown" : "keyAction",
+  },
+
+  initialize: function(){ ... },
```

Comment:
These are all great, quick question. Can you be a little more specific with which lines to delete here? Are you advocating for the removal of the tagName property, the events hash, initialize, and even the `var InputView ...` declaration?

I deleted the text about it also being a function that returns an events hash.